src/components/ViewerTitle.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+import { Info } from './Icons';
+import { ModelInfo } from '../constants';
+
+import '../../css/viewer_title.css';
+interface ViewerTitleProps {
+  modelInfo: ModelInfo;
+}
+
+const ViewerTitle: React.FunctionComponent<ViewerTitleProps> = (
```

Comment:
I think a better name for this would be `ModelDisplayData` or something, since it's a lot more than just the title